singleapplication.h

```diff
@@ -124,14 +124,27 @@ class SingleApplication : public QAPPLICATION_CLASS
      */
     QString currentUser() const;
 
+    /**
+     * @brief Mode of operation of sendMessage.
+     * @enum
+     */
+    enum SendMode {
+        /**
+         * Should the sendMessage call block until the primary instance is terminated?
+         */
+        BlockingSend = 1 << 0,
+    };
+    Q_DECLARE_FLAGS(SendOptions, SendMode)
```

Comment:
I don't think we will have more options, so I would turn it into a simple enum:
```suggestion
    enum SendMode {
        Blocking,  /** Wait until the primary instance is terminated */
        NonBlocking,  /** Do not wait for the primary instance termination and return immediately */
    };
```

Reply:
I recommend the `NonBlocking` default goes first so it's code is zero.
```suggestion
    enum SendMode {
        NonBlocking,  /** Do not wait for the primary instance termination and return immediately */
        Blocking,  /** Wait until the primary instance is terminated */
    };
```

Reply:
Agree!